portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,21 +20,46 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.List;
 import java.lang.String;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
 import com.google.gson.Gson;
+// import com.google.sps.data.Task;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/form-submission")
```

Comment:
Maybe we can use /comments instead of /form-submission as that would describe the purpose of the servlet better. Renaming the servlet to fit its purpose is also important.